data/lua/core/mathx.lua

```diff
@@ -1,6 +1,10 @@
 --[========[Additional mathematical functions]========]
 print("Loading mathx module...")
 
+---Pick a random choice from a list of value_second
+---@param possible_values string|table
```

Comment:
I wouldn't have guessed that the string could contain ranges from this description.